### PR TITLE
Name change XCG --> WinUI 2 Gallery

### PR DESCRIPTION
**Pull request by gregwoo-microsoft**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/ListViewPage.xaml [hunk 1]
```diff
@@ -199,7 +199,7 @@
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
 &lt;!-- Full code to implement dragging and dropping between lists can be found in the 
-source code for the XAML Controls Gallery. -->
```
Comment (bpulliam) [inline XamlControlsGallery/ControlPages/ListViewPage.xaml:202]: WinUI Gallery

File: XamlControlsGallery/ControlPages/RevealFocusPage.xaml [hunk 2]
```diff
@@ -98,7 +98,7 @@
         </local:ControlExample>
 
         <TextBlock Margin="0,16,0,0" TextWrapping="WrapWholeWords">
-            <Bold>NOTE:</Bold> Choosing the <Bold>Reveal Focus</Bold> radio button will change the focus visuals that "UseSystemFocusVisuals" in the entire XAML Controls Gallery app.
```
Comment (bpulliam) [inline XamlControlsGallery/ControlPages/RevealFocusPage.xaml:101]: WinUI Gallery

File: XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample2_xaml.txt [hunk 3]
```diff
@@ -42,5 +42,5 @@ of MyDataTemplateSelector are below: -->
 <!-- The ItemsSource for this ItemsRepeater is a list of integers called NumberedItems.-->
                     
 <!-- ActivityFeedLayout is a custom designed virtualizing layout that loads images only as you come 
-accross them, defined in the code-behind. View the XAML Controls Gallery source code to see more
```
Comment (bpulliam) [inline XamlControlsGallery/ControlPagesSampleCode/ItemsRepeater/ItemsRepeaterSample2_xaml.txt:45]: WinUI Gallery

File: XamlControlsGallery/Debug.appxmanifest [hunk 4]
```diff
@@ -27,8 +27,8 @@
       </uap:VisualElements>
       <Extensions>
         <uap:Extension Category="windows.protocol">
-          <uap:Protocol Name="xamlcontrolsgallery">
-            <uap:DisplayName>XAML Controls Gallery (Debug)</uap:DisplayName>
+          <uap:Protocol Name="winui2gallery">
```
Comment (bpulliam) [inline XamlControlsGallery/Debug.appxmanifest:30]: winuigallery

File: XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs [hunk 5]
```diff
@@ -31,13 +31,13 @@ public static bool ShowCopyLinkTeachingTip
 
         public static void Copy(ControlInfoDataItem item)
         {
-            var uri = new Uri($"xamlcontrolsgallery://item/{item.UniqueId}", UriKind.Absolute);
+            var uri = new Uri($"winui2gallery://item/{item.UniqueId}", UriKind.Absolute);
```
Comment (bpulliam) [inline XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs:34]: winuigallery

File: XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs [hunk 6]
```diff
@@ -31,13 +31,13 @@ public static bool ShowCopyLinkTeachingTip
...
             ProtocolActivationClipboardHelper.Copy(uri, $"{Package.Current.DisplayName} - {item.Title} Sample");
         }
 
         public static void Copy(ControlInfoDataGroup group)
         {
-            var uri = new Uri($"xamlcontrolsgallery://category/{group.UniqueId}", UriKind.Absolute);
+            var uri = new Uri($"winui2gallery://category/{group.UniqueId}", UriKind.Absolute);
```
Comment (bpulliam) [inline XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs:40]: winuigallery

File: XamlControlsGallery/ItemPage.xaml.cs [hunk 7]
```diff
@@ -168,7 +168,7 @@ protected override async void OnNavigatedTo(NavigationEventArgs e)
                 {
                     // Pagetype is not null!
                     // So lets generate the github links and set them!
-                    var gitHubBaseURI = "https://github.com/microsoft/Xaml-Controls-Gallery/tree/master/XamlControlsGallery/ControlPages/";
+                    var gitHubBaseURI = "https://github.com/microsoft/WinUI-Gallery/tree/master/XamlControlsGallery/ControlPages/";
```
Comment (bpulliam) [inline XamlControlsGallery/ItemPage.xaml.cs:171]: master->winui2
/WinUIGallery/

File: XamlControlsGallery/Package.appxmanifest [hunk 8]
```diff
@@ -27,8 +27,8 @@
       </uap:VisualElements>
       <Extensions>
         <uap:Extension Category="windows.protocol">
-          <uap:Protocol Name="xamlcontrolsgallery">
-            <uap:DisplayName>XAML Controls Gallery</uap:DisplayName>
+          <uap:Protocol Name="winui2gallery">
```
Comment (bpulliam) [inline XamlControlsGallery/Package.appxmanifest:30]: winuigallery

File: XamlControlsGallery/SettingsPage.xaml [hunk 9]
```diff
@@ -116,19 +116,19 @@
                     Text="Found a bug? Want a new sample?" />
                 <RichTextBlock MaxWidth="840" Margin="0,10,0,0" IsTextSelectionEnabled="True">
                     <Paragraph>
-                        If you have found a bug within the Xaml Controls Gallery, please file an issue on GitHub:
+                        If you have found a bug within the WinUI 2 Gallery, please file an issue on GitHub:
```
Comment (bpulliam) [inline XamlControlsGallery/SettingsPage.xaml:119]: WinUI Gallery

**Review by bpulliam — CHANGES_REQUESTED** [review]

File: XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs (hunk 5 above)
Comment (gregwoo-microsoft) [reply XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs:34]: Isn't that going to be problematic if we keep the same URI for both the WinUI 2 & WinUI 3 Gallery?

Comment (bpulliam) [reply XamlControlsGallery/Helper/ProtocolActivationClipboardHelper.cs]: It means we can share test infrastructure.  They weren't different before, so we wouldn't be breaking anything. We're keeping the status quo.

**Review by bpulliam — APPROVED** [review]

**Pull request merged**